src/stdlib.rs

```diff
@@ -1,12 +1,28 @@
-use include_dir::{include_dir, Dir};
-pub const STDLIB: Dir = include_dir!("src/std");
+use std::{env, fs, path::PathBuf};
+
+#[cfg(not(debug_assertions))]
+fn get_install_dir() -> PathBuf {
```

Comment:
I don't see why you need different versions of this function for debug and release builds. Consider picking one and using it for both.

Reply:
Also, please move private functions below public function `resolve()`, which they support.